internal/client/mock.go

```diff
@@ -46,7 +46,7 @@ type Mock struct {
 	SyncedFork           bool
 }
 
-func (c *Mock) SyncFork(ctx *context.Context, head Repo, base Repo) error {
+func (c *Mock) SyncFork(_ *context.Context, _ Repo, _ Repo) error {
```

Comment:
```suggestion
func (c *Mock) SyncFork(*context.Context, Repo, Repo) error {
```

Reply:
Hi, Carlos, should i open a new PR to change it to this suggested change?

Reply:
nah, its good as is :)

Reply:
okay, thank you :)